flutter_common/lib/data/generation_helpers/write_json_example.main.dart

```diff
@@ -28,7 +28,10 @@ const fileNameEnv = 'jsonFileName';
 Future<void> main() async {
   final runResult = BenchmarkRunResult(
     throughput: 123.45,
-    accuracy: 123.45,
+    accuracy: AccuracyData(
+      value: 123.45,
```

Comment:
Should the value not be normalized?

Reply:
It doesn't matter here, we only need this to mark that the value is numeric

Reply:
It would be nice if the example value match the description of the key.

Reply:
Done